debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/oracle/generated/PlSqlParser.g4

```diff
@@ -5210,7 +5210,7 @@ object_privilege
     | DEBUG
     | DELETE
     | EXECUTE
-    | FLASHBACK ARCHIVE
+    | FLASHBACK ARCHIVE?
```

Comment:
```suggestion
    | FLASHBACK
    | FLASHBACK ARCHIVE
```

Reply:
fixed in 8958aa9bcb726882a460909e029fb3091bde087a